cheatsheets/Server_Side_Request_Forgery_Prevention_Cheat_Sheet.md

```diff
@@ -20,6 +22,27 @@ SSRF is an attack vector that abuses an application to interact with the interna
 * SSRF is not limited to the HTTP protocol, despite the fact that in general the first request leverages it, yet the second request is performed by the application itself, and thus it could be using different protocols (*e.g.* FTP, SMB, SMTP, etc.) and schemes (*e.g.* `file://`, `phar://`, `gopher://`, `data://`, `dict://`, etc.). The protocol/scheme usage is highly dependent on the application's requirements.
 * If the application is vulnerable to [XML eXternal Entity (XXE) injection](https://portswigger.net/web-security/xxe) then it can by exploited to perform a [SSRF attack](https://portswigger.net/web-security/xxe#exploiting-xxe-to-perform-ssrf-attacks), take a look at the [XXE cheat sheet](XML_External_Entity_Prevention_Cheat_Sheet.md) to learn how to prevent the exposure to XXE.
 
+## Contents
+
+- [Cases](#cases)
+  * [Case 1 - Application can send request only to identified and trusted applications](#case-1---application-can-send-request-only-to-identified-and-trusted-applications)
```

Comment:
Looks like you used the same generator I did. Not sure why it uses three different symbols `-`, `*` and `+` but I think it's better to just have them all as `-`